packages/base/src/util/TabbableElements.js

```diff
@@ -22,12 +19,10 @@ const getTabbables = (nodes, tabbables) => {
 		}
 
 		if (currentNode.shadowRoot) {
-			// If the current node has more than 1 child, the 1st child is the style tag,
-			// and the 2nd - the root node of the shadow DOM.
-			// If the current node has one child (adopted stylesheets in use),
-			// this has to be the root node of the shadow DOM.
+			// get the root node of the ShadowDom (1st none style tag)
 			const children = currentNode.shadowRoot.children;
-			currentNode = children.length > 1 ? children[1] : children[0];
+			const nonStyleTags = Array.from(children).filter(node => node.tagName !== "STYLE");
+			currentNode = nonStyleTags.length ? nonStyleTags[0] : null;
```

Comment:
the null assignment will fail in the `currentNode.tagName === "SLOT"` check below. It is guaranteed that we have a node, right? So let's not check for length.

The other possibility is to do the below checks for all `nonStyleTags` in a loop, so it will handle 0 and 2+ as well.